aten/src/ATen/native/native_functions.yaml

```diff
@@ -6484,6 +6484,32 @@
     SparseCPU, SparseCUDA: norm_sparse
   autogen: native_norm.ScalarOpt_dim_dtype_out
 
+- func: _batch_norm_with_update(Tensor input, Tensor? weight, Tensor? bias, Tensor(a!) running_mean, Tensor(b!) running_var, float momentum, float eps) -> (Tensor, Tensor, Tensor, Tensor)
+  dispatch:
```

Comment:
We should check if this has the same problem as https://github.com/pytorch/pytorch/issues/120622

Reply:
Looks like this is also an issue with this new op. @bdhirsh @albanD and I discussed this briefly and decided to fix this in a separate PR (but before we switch to the new batch norm stack)